src/commands/ocrClickOnText.ts

```diff
@@ -5,6 +5,7 @@ import { Rectangles, ScreenSize } from '../typings/types'
 interface OcrClickOnTextOptions {
   androidRectangles?:Rectangles;
   iOSRectangles?:Rectangles;
+  tesseractLang?: string;
```

Comment:
Hi @gromanas 

I've been thinking about this again, and I made a mistake. All command-file must have this as a mandatory option, so `tesseractLang?: string;` needs to be `tesseractLang: string;`. This also means that all tests that you've added (for the optional ones) are not needed and need to be removed from this PR.

The tests need to be added to every command in the `service.test.ts`-file.

Again, sorry for not seeing this the first time and causing you extra work (but hopefully you learned from it 😆 )